### PR TITLE
Fixes #9061: pinned rack version to < 1.6

### PR DESCRIPTION
**Pull request by dmitri-d**
This is a workaround until newer version of Sinatra is released.

### CODE REVIEW

**Comment by dmitri-d** [comment]
@domcleal: ping?

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ source 'http://rubygems.org'
 
 gem 'json'
 gem 'sinatra'
-gem 'rack', '>= 1.1'
```
Comment (domcleal) [inline Gemfile:5]: can we keep this too?

**Comment by dmitri-d** [comment]
@domcleal: sure thing!

**Comment by dmitri-d** [comment]
@domcleal: ping

**Comment by domcleal** [comment]
Thanks @witlessbird, merged as db28993 and I filed http://projects.theforeman.org/issues/9063 to revert it in the future.

**Comment by lazyfrosch** [comment]
+1 Tested for PR #246

**Pull request closed without merging**